FlowCrypt/Functionality/Mail Provider/Backup Provider/Imap+Backup.swift

```diff
@@ -109,7 +109,7 @@ extension Imap: BackupApiClient {
             MCOIMAPSearchExpression.search(from: email),
             other: MCOIMAPSearchExpression.search(to: email)
         )
-        return MCOIMAPSearchExpression.searchAnd(fromToExpr, other: try subjectsExpr())
+        return try MCOIMAPSearchExpression.searchAnd(fromToExpr, other: subjectsExpr())
```

Comment:
Just interesting. Why have you decided to switch to such a format?

Reply:
I mean using `try` outside

Reply:
We decided to use `swiftformat` for consistent formatting and it has default rule for such cases - [hoistTry](https://github.com/nicklockwood/SwiftFormat/blob/main/Rules.md#hoistTry).
I think moving `try` outside makes it easier to see which expressions can throw